File: JavaScript/프로그래머스/0/181922. 수열과 구간 쿼리 4/수열과 구간 쿼리 4.js
function solution(arr, queries) {
    let answer = [];
    
    for (const q of queries) {
        const s = q[0];
        const e = q[1];
        const k = q[2];
        
        for (let i=s; i<=e; i++){
            if (i%k === 0) {
                arr[i] += 1;
            }
        }
    }
    
    return arr;
}

File: C++/프로그래머스/0/181922. 수열과 구간 쿼리 4/수열과 구간 쿼리 4.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> queries) {
    for (int i = 0; i < queries.size(); i++){
        vector<int> query = queries[i];
        int s = query[0];
        int e = query[1];
        int k = query[2];
        
        for (int j=s; j<=e; j++){
            if (j%k == 0) {arr[j] += 1;}
        }
    }
    return arr;
}